docs/tutorials/data_management.md

```diff
@@ -0,0 +1,243 @@
+# Data management in ICU4X
+
+Unless you're happy shipping your app with the ~10 locales supported by `icu_testdata` (and with the size impact of all keys), you will need generate your own data that is customized to suit your needs.
+
+This tutorial introduces data providers beyond `icu_testdata`, as well as the `icu_datagen` tool.
+
+# 1. Prerequesites
+
+This tutorial assumes you have finished the [introductory tutorial](intro.md) and continues where that tutorial left off. In particular, you should still have the latest version of code for `myapp`.
+
+# 2. Generating data
+
+Data generation is done using the `icu_datagen` crate, which pulls in data from [Unicode's *Common Locale Data Repository* (*CLDR*)](http://cldr.unicode.org/index/downloads) and from `ICU4C` releases to generate `ICU4X` data. The crate has a command line interface as well as a Rust API, which can be used for example in `build.rs` files. Here we will use the CLI.
+
+First we will need to install the binary:
+
+```console
+$ cargo install icu_datagen --features bin
+```
+
+Get a coffee, this might take a while ☕.
+
+Once installed, run:
+
+```console
+$ icu4x-datagen --cldr-tag latest --icuexport-tag latest --out my-data --format blob --all-keys --all-locales
+```
+
+Let's dissect this invocation:
+* `--cldr-tag` selects the CLDR version to use
+* `--icuexport-tag` selects the ICU-exported data version to use
+* `--out` is the location where we want the generated ICU4X data to be stored
+* `--format` sets the format of the output (we'll discuss formats later)
+* `--all-keys` `--all-locales` specifies that we want to include all data for all locales
```

Comment:
Suggestion: add "This will generate a lot of data, which we will optimize later in the tutorial."

Reply:
Given that this is already merged, please make a PR with your proposed changes.